src/style.css

```diff
@@ -1 +1,72 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
```

Comment:
The font-weight is set to 300 for Avenir-Book. Ensure that this matches the actual weight of the font file. If the font file is not of weight 300, it might not render as expected.